libsemigroups_pybind11/todd_coxeter.py

```diff
@@ -6,14 +6,16 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 
-# pylint: disable=no-name-in-module, invalid-name, missing-function-docstring
-# pylint: disable=unused-import, missing-module-docstring, protected-access
-# pylint: disable=duplicate-code
+"""
+This page contains the documentation for various helper functions for
+manipulating :any:`ToddCoxeter` objects. All such functions
+are contained in the subpackage ``todd_coxeter``.
+"""
 
-from typing import Union, List, Iterator
-from typing_extensions import Self
+from typing import List
+from typing_extensions import Self as _Self
 
-from _libsemigroups_pybind11 import (
+from _libsemigroups_pybind11 import (  # pylint: disable=no-name-in-module,unused-import
```

Comment:
We can get rid of `unused-imports` here if we also remove `PositiveInfinity`, `_PresentationStrings`, `_PresentationWords` and `Undefined`.